tests/SprykerTest/Zed/EventBehavior/Business/EventBehaviorFacadeTest.php

```diff
@@ -116,6 +119,122 @@ function ($eventName, TransferInterface $eventTransfer) {
         $this->eventBehaviorFacade->triggerLostEvents();
     }
 
+    /**
+     * @return void
+     */
+    public function testExecuteResolvedPluginsBySources(): void
+    {
+        $behaviorStatus = Config::get(EventBehaviorConstants::EVENT_BEHAVIOR_TRIGGERING_ACTIVE, false);
+        if (!$behaviorStatus) {
+            return;
+        }
+
+        $this->createEntityChangeEvent();
+
+        $container = new Container();
```

Comment:
It would be better to prepare the container in a separate function

Reply:
Did separate function for container preparation.